packages/select/src/components/query-list/queryList.tsx

```diff
@@ -34,6 +34,12 @@ import {
 export type QueryListProps<T> = IQueryListProps<T>;
 /** @deprecated use QueryListProps */
 export interface IQueryListProps<T> extends ListItemsProps<T> {
+    /** If provided, this function will be used in place of the default implementation. */
```

Comment:
I borrowed the wording from https://github.com/palantir/blueprint/blob/b6c1518ea2f524accfba8045ce6f22cac7c5b56e/packages/core/src/context/hotkeys/hotkeysProvider.tsx#L98